modules/core/src/main/java/org/apache/synapse/config/SynapseConfigUtils.java

```diff
@@ -895,6 +897,23 @@ public static boolean isFailSafeEnabled(String componentName) {
         return false;
     }
 
+    public static String processString(String input, Pattern pattern, MessageContext messageContext) {
```

Comment:
Shall we add a meaningful name to this method depending on the functionality. Better to add a javadoc comment as well.